core/controllers/practice_sessions.py

```diff
@@ -37,6 +41,28 @@ def get(self, topic_name):
             raise self.PageNotFoundException(
                 Exception('The topic with the given name doesn\'t exist.'))
 
+        interaction_ids = feconf.ALLOWED_QUESTION_INTERACTION_IDS
+
+        interaction_dependency_ids = (
+            interaction_registry.Registry.get_deduplicated_dependency_ids(
+                interaction_ids))
+        dependencies_html, additional_angular_modules = (
+            dependency_registry.Registry.get_deps_html_and_angular_modules(
+                interaction_dependency_ids))
+
+        interaction_templates = (
+            interaction_registry.Registry.get_interaction_html(
+                interaction_ids))
+
+        self.values.update({
+            'DEFAULT_OBJECT_VALUES': obj_services.get_default_object_values(),
```

Comment:
I think #6636 is trying to remove this, so this might need to be changed once that goes in.

Reply:
Should I do it now itself, or is it okay as a follow up ?

Reply:
Since the other is not merged yet, I'd say let it be. A follow up would be enough.